raft/src/test/java/org/apache/kafka/raft/KafkaRaftClientTest.java

```diff
@@ -144,6 +204,119 @@ public void testInitializeAsCandidateAndBecomeLeader() throws Exception {
             record.key(), record.value());
     }
 
+    @Test
+    public void testHandleBeginQuorumRequest() throws Exception {
+        int otherNodeId = 1;
+        int votedCandidateEpoch = 2;
+        Set<Integer> voters = Utils.mkSet(localId, otherNodeId);
+
+        MockQuorumStateStore electionStore = new MockQuorumStateStore() {
```

Comment:
Not sure why we need to extend this. Couldn't we initialize the state the same way as in other test cases? There are a few of these.